public/app/features/admin/UserSessions.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { PureComponent } from 'react';
+import { UserSession } from 'app/types';
+
+interface Props {
+  sessions: UserSession[];
+  headingStyle?: string;
+  tableStyle?: string;
```

Comment:
these styles do not seem dynamic so not sure why they are sent in as props. Just to share custom margins?